catalog/mvcc/cataloger_delete_entry_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"errors"
```

Comment:
Not sure I understand the relationship between these tests and #999.  Can we rebase on top of that, so we have those tests?  Or delete test from one side if the other side is better?

Reply:
I started before 999 was opened. I just copied the test that was added in 999. Since this is the only change of 999, I think 999 can be deleted